data/nlu/nlu.md

```diff
@@ -4998,28 +4998,34 @@ data/nlu/lookups/products.txt
 
 ## synonym:stack
 - fullstack
-- Full Stack
-- Full
-- full stack
-- full staclk
-- Full stack
 - full
+- Full stack
+- full staclk
 - full framework
+- full stack
+- Full
+- Full Stack
 
 ## synonym:wit.ai
 - wit
 
 ## synonym:x
-- rasa x
-- Rasa X
-- X
 - Rasax
-- rasax
 - RASAX
+- Rasa X
+- rasa x
 - RASA X
+- X
+- rasax
 
 ## regex:greet
 - hey[^\s]*
 
 ## regex:zipcode
 - [0-9]{5}
+
+## lookup:location.txt
+  /app/git/1/data/location.txt
+
+## lookup:products.txt
+  /app/git/1/data/products.txt
```

Comment:
This looks incorrect. personally I think the name of the lookup when dumped should just be whatever comes before .txt (though we know it doesn't actually matter for featurization), but the way it dumps the file should be relative so that it works on the server as well as in the repo -- would mention that to Tobi.